src/services/mongoose/wallet-invoices.ts

```diff
@@ -49,5 +51,81 @@ export const WalletInvoicesRepository = (): IWalletInvoicesRepository => {
     }
   }
 
-  return { persist, findByPaymentHash }
+  async function* findPendingByWalletId(
```

Comment:
We have been using the `const functionName = async () => {}` style to define functions. I think we should be consistent.

Reply:
uhmm really dont know how to implement a generator function with arrow syntax, as far as I know is still a proposal